lib/Completion/Bridge/TolerantParser/CompletionContext.php

```diff
@@ -367,6 +370,22 @@ public static function promotedPropertyVisibility(Node $node): bool
         return false;
     }
 
+    public static function conditionInfix(Node $node): bool
+    {
+        // If the current node is not a variable we're at the beginning of the condition like "if (<>"
+        if ($node->getText() === '') {
+            return false;
+        }
+
+        $parent = $node->parent;
+        if (!$parent instanceof ExpressionStatement) {
+            return false;
```

Comment:
Why not like previously:
```suggestion
        if ($parent instanceof IfStatementNode || $parent instanceof WhileStatement) {
            return true;
```

Reply:
I mean technically it could occur in any expression. But this was an error in the tests. I've fixed it now.